src/core/sorted_map.cc

```diff
@@ -711,7 +711,7 @@ size_t SortedMap::LexCount(const zlexrangespec& range) const {
 }
 
 bool SortedMap::Iterate(unsigned start_rank, unsigned len, bool reverse,
-                        absl::FunctionRef<bool(sds, double)> cb) const {
+                        std::function<bool(sds, double)> cb) const {
```

Comment:
Because `absl::FunctionRef` does not work with lambdas that have a `mutable` state. Moreover, `cb` is passed to the low level interfaces which use `std::function`. 

Ps, think of `absl::FunctionRef` as a `const std::function&` but because it's `const` it can't really mutate its state

Reply:
The advantage of `absl::FunctionRef` is that no allocation is ever done (like `string_view`). But if the underlying API uses a `std::function` then there's no point in having it, so SG